tests/bson/bson-int64-operation-001.phpt

```diff
@@ -0,0 +1,72 @@
+--TEST--
+MongoDB\BSON\Int64 operations
```

Comment:
Note my previous comments on the operation handler. This file will likely get much longer if you cover the overflow/underflow cases, so you may want to split it up by operation or groups of operations.

Reply:
I split this test into several tests for various operations (arithmetic, exponentiation, bitwise operators, others).